src/game/shell.c

```diff
@@ -187,6 +189,60 @@ void Shell_Wait(int nticks)
     }
 }
 
+void Shell_ValidateLevelTitle(char *str)
+{
+    // Strip non alphanumeric chars
+    char *ix = str;
+    int idx = 0;
+    while (*ix != '\0') {
+        if ((*ix >= 'A' && *ix <= 'Z') || (*ix >= 'a' && *ix <= 'z')
+            || (*ix >= '0' && *ix <= '9') || *ix == '\'' || *ix == '.'
+            || *ix == ' ') {
+            idx++;
+            *ix++;
+        } else {
+            memmove(&str[idx], &str[idx + 1], strlen(str) - idx);
+        }
+    }
+
+    // Make sure title between 0 and LEVEL_TITLE_SIZE chars
+    if (strlen(str) == 0) {
+        memmove(str + 1, str, 2);
+        str[0] = 'x';
```

Comment:
`x`? Do we ever run into this case?

Reply:
If the user names the level title `""` in the gameflow the level title would just be a NULL string. ("title": "Caves") I decided to at least give it a one char temp name to be safe since we are checking for other bad level titles already. And `x` is like the generic algabraic symbol idk lol. I could change it to `Level X` or something.

Reply:
Does it make sense to name the screenshots `20220106_165959_x.jpg`? IMO `20220106_165959.jpg` would suffice.

Reply:
So remove the level title? I guess we could, but I kind of like having it. It's up to you though. I'm not a heavy screenshot end user, so it doesn't really matter to me.

Reply:
I mean, only remove the level title if the author failed to supply it instead of providing a dummy placeholder. I'd prefer to keep it under normal circumstances as well.

Reply:
I have an idea. I'll just call it `Level X` where `X` is the level number.